plugin/gf/user.vim

```diff
@@ -52,12 +52,18 @@ call s:define_ui_key_mappings()
 
 command! -bang -bar -nargs=0 GfUserDefaultKeyMappings
 \ call s:cmd_GfUserDefaultKeyMappings(<bang>0)
+
+let s:mapcmd = ['nmap', 'vmap']
+if exists(':xmap') == 2
```

Comment:
This branching is verbose. `:xmap` is always available because vim-gf-user requires Vim 7.3 or later.

Reply:
Line 36 must be updated too for consistency.

Reply:
Okay, This plugin's policy is...

- Vim : 7.3 or later -> xmap use OK.
- vmap using not need -> replace xmap OK.

I understood above.

next fix wait a little.

Reply:
Fix